app/components/author-select.hbs

```diff
@@ -1,6 +1,6 @@
-<select {{on "change" (fn this.change)}} class="form-control">
-  {{#each authors as |author|}}
-    <option value={{author.id}} selected={{is-equal author.id default.id}}>
+<select {{on "change" this.change}} class="form-control">
+  {{#each this.authors as |author|}}
+    <option value={{author.id}} selected={{is-equal author.id this.default.id}}>
```

Comment:
Should we replace this with [`<PowerSelect />`](https://ember-power-select.com/) or does that "take away" from the tutorial?